openedx/core/djangoapps/user_api/api.py

```diff
@@ -415,14 +415,23 @@ def _add_password_field(self, form_desc, required=True):
         # meant to hold the user's password.
         password_label = _(u"Password")
 
+        restrictions = {
+            "min_length": password_min_length(),
+            "max_length": password_max_length(),
+        }
+
+        complexities = getattr(settings, 'PASSWORD_COMPLEXITY', {})
+        if not settings.FEATURES.get('ENFORCE_PASSWORD_POLICY', False):
```

Comment:
Is this a new or existing setting?

Reply:
Existing setting and I hate it. I don't know what purpose it serves. I opened a bug about "can we drop this feature flag?" and started talking to the McKinsey team about it.